app/controllers/games_controller.rb

```diff
@@ -33,4 +37,12 @@ def set_game
     end
     redirect_to game_path(@game) if @game.srdc.try(:shortname).present? && params[:game] == @game.id.to_s
   end
+
+  def set_games
+    @game_sections = SpeedrunDotComGame.order(name: :asc).slice_when do |a, b|
+      a.name[0].downcase != b.name[0].downcase
+    end.map do |srdc_games|
```

Comment:
Rubocop does not like chaining blocks, and here I would agree.  We're creating a completely new array just to throw it away and create a hash from them.

Reply:
If you're still using vim there is a plugin(?) that will autorun rubocop.  I'm not sure what it's called since I don't use it anymore (tried and failed to move to it) but I still have it installed if you can tell me where to look.